admin/python/lsst/qserv/admin/dataloader.py

```diff
@@ -46,6 +46,7 @@
 #----------------------------------
```

Comment:
What is the reason for renaming this module? I think its original name `dataLoader` already follows DM guidelines for camelCase naming.